uwblueprint/childrens-aid-society
# Multi text input + Goals input

## Notion ticket link

[Create multi-text field component](https://www.notion.so/uwblueprintexecs/Create-multi-text-field-component-9bf15e7f71bc47a2b8423c188c0867cc)



## Implementation description
* Created a div that is an array of divs with an input field at the end to simulate a single input field with tags
* Created autocomplete to select tags
* Ability to add and delete tags with enter and delete keypresses



## Steps to test
1. go to `http://localhost:3000/intake`
2. go to the Program details
3. Enter Familial concerns
4. Entering text should behave like a text input
5. Pressing enter with text should create a tag
6. Pressing enter without text should do nothing
7. Pressing backspace while there's no text should delete the last tag
8. Pressing backspace while there is text should backspace text as usual
9. Presssing the X on a tag should delete it
10. Clicking on the text input should open the autocomplete
11. Clicking autocomplete option should add that tag to the field
12. Entering text that is not on the suggestions should show no autocomplete



## What should reviewers focus on?
* `MultiTextInput.tsx`


## Checklist
